app/lib/templates.dart

```diff
@@ -145,11 +160,7 @@ class TemplateService {
       final content = file.text;
       if (content != null) {
         if (isMarkdownFile(filename)) {
-          try {
-            return cmd.markdownToHtmlWithSyntax(content);
-          } catch (e) {
-            return cmd.markdownToHtmlWithoutSyntax(content);
-          }
+          return _markdownToHtmlWithoutSyntax(content);
```

Comment:
Well, just call it '_markdownToHtml` – since this method does no syntax highlighting. That happens on the client w/ JS